server/views/map/show.jade

```diff
@@ -1,14 +1,14 @@
 extends ../layout-wide
 block content
-        .text-center.map-fixed-header
-            p Required challenges are marked with a *
-            .row
-                .col-xs-10.col-xs-offset-1.col-sm-6.col-sm-offset-3.col-md-4.col-md-offset-4
-                  .row
-                    .col-xs-12.col-sm-8.col-sm-offset-2.col-md-6.col-md-offset-3
-                      .btn.btn-sm.btn-block.btn-primary.active#manipAll Show incomplete challenges
-                      .btn.btn-sm.btn-block.btn-primary.active#showAll Collapse all
-            hr
+    .text-center.map-fixed-header
+        p Required challenges are marked with a *
+          .row
+            .col-xs-12.col-sm-8.col-sm-offset-2.col-md-6.col-md-offset-3
+              .btn.btn-sm.btn-block.btn-primary.active#manipAll Show incomplete challenges
+              .btn.btn-sm.btn-block.btn-primary.active#showAll Collapse all
```

Comment:
I don't think you should change the button spacing at all.  Or, rather, the buttons are inside a row inside a row and they look "ok" on the current site, I think.  Can you just make the change you made below to the row that contains the accordian?  Or are they nested in some way?